nservicebus/sqlserver/multiple-databases.md

```diff
@@ -14,24 +14,48 @@ The SQL Server transport allows you to select, on per-endpoint basis, where the
 
 The transport will route messages to destination endpoints based on the configuration. If no specific configuration has been provided for a particular destination endpoint, the transport assumes the destination has the same configuration (schema, database and instance name/address) as the sending endpoint. If this assumption turns out to be false (the transport cannot connect to destination queue), an exception is thrown immediately. There is no store-and-forward mechanism on the transport level (and hence -- no dead-letter queue).
 
-NOTE: If the destination endpoint uses different database or server instance, sending a message to it might cause the transaction to escalate to a distributed transaction. Usually it is not a desired effect and one can use NServiceBus Outbox to avoid it.
+NOTE: If the destination endpoint uses different database or server instance, sending a message to it might cause the transaction to escalate to a distributed transaction. Usually it is not a desired effect. Use NServiceBus [Outbox](/nservicebus/outbox/) to avoid it.
```

Comment:
@indualagarsamy I think the note should say:
 * If input queue, error queue, outbox and business data are on the same db instance user can avoid escalating to DTC
 * If handler produces at least two messages which are destined to queues on different db instances endpoint will escalate to DTC